<a href="https://colab.research.google.com/github/JMateusNSilva/python_uerj_2020_2/blob/master/Aula_01_18_set_2020.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Aula 01 - Dia 18/ set/ 2020 \\
Prof: Dilson \\

Exercícios:

In [ ]:
print('mundo, mundo, vasto mundo')

mundo, mundo, vasto mundo


In [ ]:
help('print')

Help on built-in function print in module builtins:

print(...)
    print(value, ..., sep=' ', end='\n', file=sys.stdout, flush=False)
    
    Prints the values to a stream, or to sys.stdout by default.
    Optional keyword arguments:
    file:  a file-like object (stream); defaults to the current sys.stdout.
    sep:   string inserted between values, default a space.
    end:   string appended after the last value, default a newline.
    flush: whether to forcibly flush the stream.



**Exercício 03:** Se você fizer uma corrida de 10 Km em 43 min e 30 s, qual será a seu tempo médio por milha? Qual sua velocidade média em milha por hora?(Dica: há 1,61 quilômetros em uma milha).

**Resolvendo:** \\
Transformando o tempo em minutos

In [ ]:
43 + 30 *(1/60)

43.5

tempo igual a 43,5 minutos.

Transformando quilômetros para milhas,

In [ ]:
10*1.6

16.0

Então, a distância percorrida é de 16 milhas. \\
Calculemos agora o tempo médio percorrido por cada milha,

In [ ]:
43.5/16

2.71875

O tempo médio percorrido por cada milha é de aproximadamente 2,719 min/milha.

Agora devemos calcular qual foi a velocidade média em milhas por hora \\
\begin{equation}
v_{med}=\frac{\Delta S}{\Delta t} \qquad \text{.}
\end{equation}

Já sabemos que $\Delta S = 16$ milhas, precisamos descobrir qual foi o tempo gasto em horas. Para isto fazemos a seguinte conta,

In [ ]:
43.5/60

0.725

. Assim obtemos o valor de 0,725 horas gastas para percorrer 16 milhas. Fazendo o calculo para $v_{med}= \frac{16\text{ milhas}}{0,725\text{ h}}$,

In [ ]:
16/0.725

22.06896551724138

. Assim afirmamos que as 16 milhas foram percorridas com uma velocidade média de aproximadamente 22.07 milhas/h